tpm2/INSTALL.md

```diff
@@ -47,7 +47,7 @@ these dependencies can likely be met using your package
 manager. [libkmip](https://github.com/OpenKMIP/libkmip) must be
 installed from source.
 
-##### CentOS 7 (Red Hat 7) Commands
+##### CentOS 8 (Red Hat 8) Commands
 
 ```yum install openssl openssl-devel glibc gcc libffi-devel indent```
```

Comment:
For CentOS 8 `indent` got moved to the EPEL repo.

Reply:
The Makefile still uses indent. We should either remove it from there (and so also from the Ubuntu installation instructions) or mention you need it but it's in EPEL so you have to activate that first.